binr/r2r/r2r.c

```diff
@@ -158,6 +168,9 @@ int main(int argc, char **argv) {
 		case 'C':
 			r2r_dir = opt.arg;
 			break;
+		case 'n':
```

Comment:
this is not in the help. What does it do?

Reply:
It does nothing

Reply:
was that serious? :D

Reply:
Yes, see line 267.

Reply:
mh, ok... it should be listed in the help msg anyway though

Reply:
fixed